### ISSUE TITLE
[hi] Tracking Hindi localization progress

### ISSUE THREAD

**Issue by anubha-v-ardhan** [kubernetes/website#29353]
Summary of progress till now: https://github.com/kubernetes/website/issues/29353#issuecomment-919357503

Our current aim is to get the localization live first with [minimum required content](https://kubernetes.io/docs/contribute/localization/#minimum-required-content). We will be ready to expand the localization including other pages once bare minimum version goes live.

# Maintainers
Approvers:
- @anubha-v-ardhan 
- @divya-mohan0209 
- @mittalyashu 

Reviewers:
- @anubha-v-ardhan 
- @divya-mohan0209 
- @verma-kunal 
- @mittalyashu 
#### To translate a page:

1. Check that no one else has claimed your page in the checklist and comments below.
2. Comment below with the location of the English page you would like to translate. **Take only one page at a time.**
3. Maintainers will add your username along your task in the checklist.
4. Fork this repo, make your localisations, and submit a **pull request** following the name convention **`[hi] [Add/Update/Fix] [content/hi/file/full/path]`** and base branch as `dev-1.22-hi.1`. **Don't forget to use `/language hi` command**. Also, **tag/ping a maintainer** to make it easier to track progress and add your PR to the checklist below.
For example:
**`[hi] Add content/hi/docs/concepts/architecture/nodes.md`**
**`[hi] Update content/hi/setup/release/building-from-source.md`**

**Note: To know about the preferred localization of common Kubernetes related terms, check this [spreadsheet](https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0)**
**Note: To get an idea about the kind of language we are expecting, have a look at [this PR](https://github.com/cncf/foundation/pull/175)**

#### Points to remember:

1. You can use this script to know if a given file is touched by which PR: [**https://github.com/kubernetes/website/blob/master/scripts/find_pr.py**](https://github.com/kubernetes/website/blob/main/scripts/find_pr.py) By doing so, you can have a single source of truth about whether a file is currently being modified or not.
2. Please use the word **`localisation`** instead of **`translation`** in your PRs and comments.
3. **Every new contribution should start with a new branch from `dev-1.22-hi.1`.**
For more detailed instruction, check the [website-README](https://github.com/kubernetes/website/blob/main/README-hi.md). 
4. Before contributing to docs, read the [**Community Documentation Style Guide**](https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md) and the [**Kubernetes Documentation Style Guide**](https://kubernetes.io/docs/contribute/style/style-guide/).

**If you're still not sure about something, please don't hesitate to ask in [Slack channel](https://app.slack.com/client/T09NY5SBT/CJ14B9BDJ/thread/CDHA7K9L5-1628663075.294300).**

Please commit with your translations! If you find that you can't commit anymore, let the maintainers know so they can assign the page to someone else.

## For Maintainers

When someone volunteers, edit this issue with the username of the volunteer, and with the PR. 
Example : **`Init Hindi localisation (@mittalyashu) #13910`**
When PRs are merged, make sure to mark that page as completed!

# Website (`content/en`)
- [x] Landing (@verma-kunal)
#30087

# releases (`content/en/releases`)
>- [x] Landing (@AvineshTripathi)
#31273
#31292

# includes (`content/en/includes`)
>- [x] task-tutorial-prereqs.md (@Nawaz027)
#30590 
#30602

# case-studies (`content/en/case-studies`)
>- [x] Landing (@harshmathurx)
#30731 
#30733 

# Docs (`content/en/docs`)
- [x] Landing (@anubha-v-ardhan)  
#29373
> ## concepts (`content/en/docs/concepts`)
>> ### overview
>>>- [x] what-is-kubernetes.md (@kranurag7, @ashish-jaiswar)
#30732
#30847 
> ## contribute (`content/en/docs/contribute`)
>>- [x] Landing (@Nawaz027)
#31012 
#31162 
> ## reference (`content/en/docs/reference`)
>>- [x] Landing (@Garima-Negi)
#31011 
#31185 
>> ### glossary
>>>- [x] Landing (@Polokghosh53, @ashish-jaiswar)
#30591 
#30652 
> ## home (`content/en/docs/home`)
>>- [x] Landing (@verma-kunal) 
#29383 
>>- [x] supported-doc-versions (@verma-kunal) 
#29372
> ## setup (`content/en/docs/setup`)
>>- [x] Landing (@keshavcodex)
#29514
>> ### best-practices
>>>- [x] Landing (@anubha-v-ardhan)  
#29374
>> ### learning-environment
>>>- [x] Landing (@Darshnadas)
#29416
>> ### production-environment
>>> - [x] Landing (@keshavcodex)
#29518
>>> - [x] container-runtimes (@Darshnadas @ashish-jaiswar)
#31857
#31987
#32713
>>> - [x] turnkey-solutions (@ShivamTyagi12345) 
#30303
>>> #### windows
>>>>- [x] Landing (@anubha-v-ardhan)
#29413
> ## tasks (`content/en/docs/tasks`)
> - [x] Landing (@rajatgupta24)
#29434
>> ### tools
>>- [x] Landing (@rajatgupta24)
#29458
>>- [x] install-kubectl-linux (@vedant-kakde)
#29457
>>- [x] install-kubectl-macos (@vedant-kakde)
#29461
>>- [x] install-kubectl-windows (@vedant-kakde)
#29465
>> #### included
>>>- [x] Landing (@kaiwalyakoparkar)
#29705
>>>- [x] kubectl-convert-overview (@ShivamTyagi12345)
#29970
>>>- [x] kubectl-whats-next (@ShivamTyagi12345)
#29471
>>>- [x] optional-kubectl-configs-bash-linux (@ashish-jaiswar)
#30569
#30627
>>>- [x] optional-kubectl-configs-bash-mac (@vedant-kakde, @kranurag7)
#30570 
#30607 
>>>- [x] optional-kubectl-configs-zsh (@vedant-kakde)
#29474
>>>- [x] verify-kubectl (@vedant-kakde)
#29469
> ## tutorials (`content/en/docs/tutorials`)
>- [x] Landing (@anubha-v-ardhan) 
#29376
>- [x] hello-minikube (@anubha-v-ardhan)
#29396
>> ### kubernetes-basics
>>>- [x] Landing (@rajatgupta24)
#29391
>>> #### create-cluster
>>>>- [x] Landing (@rajatgupta24)
#29392
>>>>- [x] cluster-interactive (@rajatgupta24)
#29392
>>>>- [x] cluster-intro (@rajatgupta24)
#29392
>>> #### deploy-app
>>>>- [x] Landing (@rajatgupta24)
#29400
>>>>- [x] deploy-interactive (@rajatgupta24)
#29393
>>>>- [x] deploy-intro (@rajatgupta24)
#29401
>>> #### explore
>>>>- [x] Landing (@anubha-v-ardhan)
#29399
>>>>- [x] explore-interactive (@anubha-v-ardhan)
#29403
>>>>- [x] explore-intro (@anubha-v-ardhan)
#29412
>>> #### expose
>>>>- [x] Landing (@rajatgupta24)
#29430
>>>>- [x] expose-interactive (@rajatgupta24)
#29405
>>>>- [x] expose-intro (@rajatgupta24)
#29407
>>> #### scale
>>>>- [x] Landing (@verma-kunal)
#29421
>>>>- [x] scale-interactive (@verma-kunal)
#29410
>>>>- [x] scale-intro (@verma-kunal)
#29420
>>> #### update
>>>>- [x] Landing (@ShivamTyagi12345)
#29971
>>>>- [x] update-interactive (@verma-kunal)
#29432
>>>>- [x] update-intro (@verma-kunal)
#29454

# Site strings (`data/i8n/hi`)
>- [x] hi.toml (@prashantpandey9, @Babapool)
#29472
#31013
#31072

**Comment by verma-kunal** [comment]
Nice work @anubha-v-ardhan !! 👍

**Comment by anubha-v-ardhan** [comment]
@verma-kunal Please don't assign the issue as this is a collective effort of many people. We want new contributors to see this as an unassigned issue on `issues` tab
/unassign @verma-kunal

**Comment by verma-kunal** [comment]
alright sure!! Understood 👍

**Comment by verma-kunal** [comment]
@anubha-v-ardhan , i think it would be good to mention the source folder of all the files as well, that needs to be localised
`https://github.com/kubernetes/website/tree/main/content/en` 
this would be easier for new folks to navigate!!

**Comment by anubha-v-ardhan** [comment]
I have added location for the parent folders. For ease of new contributors, The identation of issue is as per their location in the website repo.

**Comment by verma-kunal** [comment]
Remember folks, all the changes are needed to be done here: `https://github.com/kubernetes/website/tree/main/content/hi`.
That means, whenever you assign yourself a particular page, after localising that, refer the file structure in the `source folder` (English one) & create new files/folder according to that (with the Hindi localization)

Feel free to ask any doubts you would have in this 🙂

**Comment by verma-kunal** [comment]
I would be starting from the `website/content/en/docs/setup` folder from today, whenever I'll be getting the time !! 🙂

**Comment by anubha-v-ardhan** [comment]
@verma-kunal Please tajke one file at a time. A whole directory is a lot of work. Hope you understand

**Comment by anubha-v-ardhan** [comment]
I am working on `website/content/en/docs/_index.html`

**Comment by verma-kunal** [comment]
> @verma-kunal Please take one file at a time. A whole directory is a lot of work. Hope you understand

oops, wrong location added !! Changing now

**Comment by keshavcodex** [comment]
I am starting with this folder [website/content/en/docs/setup/_inde.html](https://github.com/kubernetes/website/blob/main/content/en/docs/setup/_index.md) to localise it.

**Comment by ShivamTyagi12345** [comment]
I am working on `website\content\en\docs\setup\best-practices\node-conformance.md`

**Comment by verma-kunal** [comment]
@anubha-v-ardhan , how are we keeping the branch names (for us) !! I think we should decide a convention for that 
what do you think?

**Comment by anubha-v-ardhan** [comment]
@verma-kunal personal branch names are your own choices, feel free to do it your own way. : )

**Comment by verma-kunal** [comment]
Working on `website/content/en/docs/home/_index.md` now!!

**Comment by anubha-v-ardhan** [comment]
Now working on `content/en/docs/setup/best-practices/_index.md`

**Comment by anubha-v-ardhan** [comment]
Next I am working on `content/en/docs/setup/best-practices/certificates.md`

**Comment by sftim** [comment]
For `content/hi/docs/setup` you need to localize that page and its _direct subheadings_.
For example:
- https://kubernetes.io/docs/setup/ _needed_
- https://kubernetes.io/docs/setup/learning-environment/ _needed_ (including the redirect)
- https://kubernetes.io/docs/setup/production-environment/ _needed_
- https://kubernetes.io/docs/setup/production-environment/tools/ _eventually but **not for the initial localization**_
- https://kubernetes.io/docs/setup/best-practices/ _needed_

If you're not sure about anything, there's a localization Zoom call once each month. Folks there who have already done this for their localization can help you work out the details.

**Comment by sftim** [comment]
For tutorials, you _only_ need to localize
- https://kubernetes.io/docs/tutorials/kubernetes-basics/ (the whole tutorial - I recommend splitting this)
- https://kubernetes.io/docs/tutorials/hello-minikube/

as minimum required content.

**Comment by anubha-v-ardhan** [comment]
Thank you so much @sftim, Our initial checklist includes the files suggested by you. We will try to attend localization meetings.

**Comment by verma-kunal** [comment]
Thank you for the info @sftim !! Would follow these as we go ahead 🙂

**Comment by verma-kunal** [comment]
Hey @anubha-v-ardhan , i've made a draft PR here: https://github.com/kubernetes/website/pull/29372
Can you pls add that to the list above!!

**Comment by verma-kunal** [comment]
And we had to create a `draft PR` for now, right? not the normal one

**Comment by anubha-v-ardhan** [comment]
I have opened normal ones. I don't think it matters. We will basically change the branch once its available.

**Comment by verma-kunal** [comment]
oh, actually @sftim had mentioned this, so thats why I was asking!!
cool then

**Comment by anubha-v-ardhan** [comment]
Oh sorry, my bad. we have to open draft PR's

**Comment by verma-kunal** [comment]
> Oh sorry, my bad. we have to open draft PR's
yeah!

**Comment by ShivamTyagi12345** [comment]
would next work on` content\en\docs\setup\best-practices\enforcing-pod-security-standards.md`

**Comment by Darshnadas** [comment]
Hello I would like to work on this  `website/content/en/docs/setup/learning-environment/_index.md`

**Comment by verma-kunal** [comment]
Sure, go ahead @Darshnadas !! Welcome 🙂

**Comment by verma-kunal** [comment]
Hey @anubha-v-ardhan , made a PR for `content/en/docs/home/_index.md` here: https://github.com/kubernetes/website/pull/29383

**Comment by ShivamTyagi12345** [comment]
hey @anubha-v-ardhan ,drafted a PR for` website\content\en\docs\setup\best-practices\node-conformance.md` check: [`this`](https://github.com/kubernetes/website/compare/main...ShivamTyagi12345:hindi?expand=1)

**Comment by kodeninja** [comment]
Pure Telugu/Hindī suggestions for some core k8s terms (also commented about this in https://github.com/kubernetes/website/pull/29383#issuecomment-898721517):

`pod` `కోశము` `कोश` 
`service` `ఆయోగము` `आयोग`
`replica` `ప్రతికృతి` `प्रतिकृति`
`deployment` `వినిమయము` `विनिमय`
`orchestration` `సంపృక్తము` `संपृक्त`

Perhaps more such terms can be found.

Source: https://twitter.com/vakibs/status/1426299108596273155

**Comment by anubha-v-ardhan** [comment]
@kodeninja Thanks for the suggestions but we think the name of Kubernetes objects such as `deployment`, `pod`, `service` etc should be written as `डिप्लॉयमेंट`, `पॉड`, सर्विस` only.
We would like to hear your views @mittalyashu

**Comment by verma-kunal** [comment]
@anubha-v-ardhan , would be working on this `content/en/docs/setup/best-practices/cluster-large.md` now !!

**Comment by rajatgupta24** [comment]
Hey @anubha-v-ardhan, I'd like to work on `content/en/docs/tutorials/kubernetes-basics/_index.html`

**Comment by rajatgupta24** [comment]
Hey @anubha-v-ardhan, now I'm working on content/hi/docs/tutorial/kubernetes-basics/create-cluster/_index.html

**Comment by mittalyashu** [comment]
> @kodeninja Thanks for the suggestions but we think the name of Kubernetes objects such as `deployment`, `pod`, `service` etc should be written as `डिप्लॉयमेंट`, `पॉड`, `सर्विस` etc only.
> The aim of this localization is not to have pure word to word translation of all the terms but to have the docs in Hindi for easy understanding. Hope you understand
> We would like to hear your views @mittalyashu

I agree with @anubha-v-ardhan

**Comment by anubha-v-ardhan** [comment]
Sure @mittalyashu . I will

**Comment by mittalyashu** [comment]
I am don't remember how many pages I localised last time and how many of them got updated.

Can we go one file per PR? It will be easy for me to review as well.

**Comment by anubha-v-ardhan** [comment]
> I am don't remember how many pages I localised last time and how many of them got updated.
> 
> Can we go one file per PR? It will be easy for me to review as well.

cc @rajatgupta24

**Comment by rajatgupta24** [comment]
Sure, ok

> > I am don't remember how many pages I localised last time and how many of them got updated.
> > Can we go one file per PR? It will be easy for me to review as well.
> 
> cc @rajatgupta24

**Comment by rajatgupta24** [comment]
Hey @anubha-v-ardhan, now I'm working on `content/hi/docs/tutorial/kubernetes-basics/deploy-app/deploy-interactive.html`

**Comment by verma-kunal** [comment]
adding on to @mittalyashu 's point about PRs, try to follow the naming convention as mentioned in the above note, so that it would be easier to locate & review!!
For example:
`Add content/hi/docs/concepts/architecture/nodes.md`
`Update content/hi/setup/release/building-from-source.md`

cc @rajatgupta24

**Comment by verma-kunal** [comment]
Hey, actually i had a doubt!
If we for example take the translation of the word `Nodes`: originally it is `नोड्स `, but if we wanna go with the pronunciation as well, then it should be `नोड्ज़ ` (for the English word `Nodez`).
Which one is better to be considered in this situation? & then we can add that to the [sheet](https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0) as well.

**Comment by anubha-v-ardhan** [comment]
I think `नोड्स` fit better

**Comment by rajatgupta24** [comment]
> Hey, actually i had a doubt!
> If we for example take the translation of the word `Nodes`: originally it is `नोड्स `, but if we wanna go with the pronunciation as well, then it should be `नोड्ज़ ` (for the English word `Nodez`).
> Which one is better to be considered in this situation? & then we can add that to the [sheet](https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0) as well.

I think नोड्ज़, because it's the right pronunciation

**Comment by rajatgupta24** [comment]
Hey @anubha-v-ardhan, now I'm working on content/hi/docs/tutorial/kubernetes-basics/deploy-app/landing

**Comment by rajatgupta24** [comment]
Hey @anubha-v-ardhan, now I'm working on `content/hi/docs/tutorial/kubernetes-basics/deploy-app/deploy-intro.html`

**Comment by keshavcodex** [comment]
i am starting with `website\content\en\docs\setup\best-practices\multiple-zones.md`

**Comment by rajatgupta24** [comment]
@anubha-v-ardhan, I'm working on `content/hi/docs/tutorial/kubernetes-basics/expose/expose-interactive.html`

**Comment by Darshnadas** [comment]
@anubha-v-ardhan I am working on this page https://github.com/kubernetes/website/blob/main/content/en/docs/setup/learning-environment/_index.md

**Comment by rajatgupta24** [comment]
@anubha-v-ardhan, I'm working on content/hi/docs/tutorial/kubernetes-basics/expose/expose-intro.html

**Comment by rajatgupta24** [comment]
@anubha-v-ardhan, I'm working on `content/hi/docs/tutorial/kubernetes-basics/expose/landing`

**Comment by verma-kunal** [comment]
@anubha-v-ardhan , now working on `content/en/docs/tutorials/kubernetes-basics/expose/_index.md`

**Comment by anubha-v-ardhan** [comment]
hey @verma-kunal The file has already been reserved by @rajatgupta24

**Comment by verma-kunal** [comment]
@anubha-v-ardhan , taking up `docs/tutorials/kubernetes-basics/scale/scale-interactive.html`

**Comment by verma-kunal** [comment]
@anubha-v-ardhan , working on `docs/tutorials/kubernetes-basics/scale/scale-intro.html` now !!

**Comment by ShivamTyagi12345** [comment]
drafted my PR #29414 .  Review appreciated. Would work on `content\en\docs\setup\production-environment\turnkey-solutions.md` next.
 thanks

**Comment by sftim** [comment]
My tip: if you see Nodes (with a capital N), leave it as Nodes. The Kubernetes API knows about Nodes but not about any localized term. You can also mention that Node is नोड in Hindi.

Compare that to the phrase: “…control plane detects failed nodes and notifies controllers…” where it might be more reasonable to write नोड or some related term and not write node at all.

Imagine you come across a page that's 100% devnagri and it mentions a Kubernetes API you have never heard of - maybe नेटवर्क नीति? You read the page and you know lots about the new API you're learning but you don't know what `kind` you would see it described as in the API. A bit of Latin text in that page would have made things much more clear.

**Comment by verma-kunal** [comment]
@anubha-v-ardhan , working on `content/en/docs/tutorials/kubernetes-basics/scale/_index.md` now

**Comment by verma-kunal** [comment]
@anubha-v-ardhan , would work on `content/en/docs/tutorials/kubernetes-basics/update/update-interactive.html` now!!

**Comment by ShivamTyagi12345** [comment]
hey made my PR #29424 . Would next work on `content\en\docs\tasks\tools\included\kubectl-convert-overview.md`

**Comment by rajatgupta24** [comment]
hey @anubha-v-ardhan, I'm working on `content/en/docs/tasks/tools/landing`

**Comment by verma-kunal** [comment]
@anubha-v-ardhan , would be working on `content/en/docs/tutorials/kubernetes-basics/update/update-intro.html` next !!

**Comment by rajatgupta24** [comment]
hey @anubha-v-ardhan, I'm working on `content/en/docs/tasks/tools/landing`

**Comment by keshavcodex** [comment]
i am taking `content/en/docs/setup/production-environment` and it is a very long page so i will be taking more time.

**Comment by anubha-v-ardhan** [comment]
@keshavcodex Please avoid taking whole directory. We like folks to take one page at a time

**Comment by keshavcodex** [comment]
> @keshavcodex Please avoid taking whole directory. We like folks to take one page at a time

ohh sorry i missed to mention the file name.

**Comment by anubha-v-ardhan** [comment]
Oh I'm very sorry @keshavcodex ! I didn't mean to dislike, I just mark pages I have assigned and not assigned with thumbs up and down for ease. Sorry

**Comment by ShivamTyagi12345** [comment]
hey made my PR #29449 .Would Next work on` \content\en\docs\tutorials\kubernetes-basics\update\_index.md`

**Comment by vedant-kakde** [comment]
Hey @anubha-v-ardhan I'm working on `content/en/docs/tasks/tools/install-kubectl-linux.md`

**Comment by kaiwalyakoparkar** [comment]
Hey how can I help in this issue?

**Comment by ShivamTyagi12345** [comment]
next work on` content\en\docs\tasks\tools\included\kubectl-whats-next.md`

**Comment by ShivamTyagi12345** [comment]
> Hey how can I help with this issue?
Hey @kaiwalyakoparkar,As you can see at the top,there are a bunch of Pages needed to be localized to Hindi,Select the ones that haven't been chosen yet by anyone and Draft your PR

**Comment by kaiwalyakoparkar** [comment]
@ShivamTyagi12345 Thanks for the reply. Where should I draft a PR btw?

**Comment by ShivamTyagi12345** [comment]
> @ShivamTyagi12345 Thanks for the reply. Where should I draft a PR btw?

Once you do the local changes in your branch , You can go ahead and choose this option:
![Screenshot (235)](https://user-images.githubusercontent.com/60812924/129872698-065b24db-d394-462c-8036-a3b69fc8dd6c.png)

**Comment by verma-kunal** [comment]
> @ShivamTyagi12345 Thanks for the reply. Where should I draft a PR btw?

you've got to create a draft PR to the main branch, as we don't have a separate branch for Hindi Localization as of yet!!

**Comment by verma-kunal** [comment]
Hey @anubha-v-ardhan , working on `docs/setup/production-environment/tools/kubeadm/_index.md` now !!

**Comment by anubha-v-ardhan** [comment]
@verma-kunal I would like you to take some other file as `docs/setup/production-environment/tools` is not required for initial localization

**Comment by kaiwalyakoparkar** [comment]
@anubha-v-ardhan I would like to work on `docs/tasks/tools/included/_index.md`

**Comment by verma-kunal** [comment]
Hey @anubha-v-ardhan , would working on this one `content/en/_index.html` be fine?

**Comment by vedant-kakde** [comment]
Hey I made my PR #29457 . Now I'm working on `content/en/docs/tasks/tools/install-kubectl-macos.md`

**Comment by rajatgupta24** [comment]
I'll start working on this file: `content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux`

**Comment by vedant-kakde** [comment]
@rajatgupta24 I think you haven't marked it as draft PR.

**Comment by vedant-kakde** [comment]
Hey I made my PR #29461 .  Now I'm working on `content/hi/docs/tasks/tools/install-kubectl-windows.md`

**Comment by arsheelsheikh** [comment]
Hey, I would like to work on this - content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md

**Comment by vedant-kakde** [comment]
Hey I made my #29465 . Now I'm working on `content/hi/docs/tasks/tools/included/verify-kubectl.md`

**Comment by arsheelsheikh** [comment]
Hey, I would like to work on this- #content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md

**Comment by arsheelsheikh** [comment]
Hi, I would like to work on this `content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md`

**Comment by vedant-kakde** [comment]
Hey I made my PR #29469 . Now I'm working on `content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md`

**Comment by vedant-kakde** [comment]
Hey @rajatgupta24 I reviewed your PR #29458 . I have suggested some changes add those in batch and commit them. 👍🏻

**Comment by anubha-v-ardhan** [comment]
@vedant-kakde You can transliterate words like kind kubeadm kubectl (if possible), just add Latin version in brackets.

**Comment by vedant-kakde** [comment]
@anubha-v-ardhan I think better we don't transliterate those words.

**Comment by anubha-v-ardhan** [comment]
@vedant-kakde It would be better if our readers know how to pronounce them while also knowing how it looks like in English.
Whenever possible we try to keep content in Devnagri with Latin versions written in brackets.

**Comment by prashantpandey9** [comment]
Site strings (data/i8n/hi)
hi.toml

can I work on this?

**Comment by ShivamTyagi12345** [comment]
Made my PR would next work on `content\en\docs\tasks\tools\included\optional-kubectl-configs-bash-mac.md`

**Comment by ShivamTyagi12345** [comment]
> Site strings (data/i8n/hi)
> hi.toml
> 
> can I work on this?

@anubha-v-ardhan  Are you sure we need to localize  (data/i8n/hi)

**Comment by anubha-v-ardhan** [comment]
@prashatpandey9 Sure, go ahead

@ShivamTyagi12345 Yes we need that

**Comment by prashantpandey9** [comment]
@anubha-v-ardhan ok thanks

**Comment by vedant-kakde** [comment]
Hey @prashantpandey9 I think you have edited the `en` doc. You need to create separate file for `hi` .  Also sign CLA agreement to be able to contribute to K8s projects.

**Comment by anubha-v-ardhan** [comment]
@vedant-kakde Nope its correct. The `en` doc was already there.

**Comment by prashantpandey9** [comment]
@vedant-kakde I did it right.

**Comment by vedant-kakde** [comment]
ok @anubha-v-ardhan @prashantpandey9

**Comment by vedant-kakde** [comment]
@prashantpandey9 for CLA agreement:- https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by prashantpandey9** [comment]
@vedant-kakde  thanks I signed it.

**Comment by kaiwalyakoparkar** [comment]
I see all the tasks taken over. Which should I work on next :sweat_smile: ?

**Comment by vedant-kakde** [comment]
@kaiwalyakoparkar don't worry my friend one left for you. 😉
`content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes`
Work on this one. 👍🏻

**Comment by kaiwalyakoparkar** [comment]
Cool @anubha-v-ardhan I would like to work on `content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes` next :slightly_smiling_face: 
Thanks @vedant-kakde ig I missed that one :+1:

**Comment by sftim** [comment]
Remember, `/docs/setup/production-environment/windows/intro-windows-in-kubernetes/` is _not_ minimum required content.

**Comment by sftim** [comment]
If folks don't see a page to localize from the minimum required content, you can still:

- work on strings localization
- offer (informal or formal) [reviews](https://kubernetes.io/docs/contribute/review/reviewing-prs/) on existing PRs
   - you can [check out the other contributor's work](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally) then [preview the change locally](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#preview-locally) and make sure that Hugo can render it OK
- help out with co-ordinating the remaining tasks to get Hindi added as an official localization

**Comment by anubha-v-ardhan** [comment]
@sftim I think we have completed localizing minimum requirements (reviewing is left). Can you check in the issue description's checklist if we missed any minimum required page. That would be great help.

**Comment by kaiwalyakoparkar** [comment]
> Remember, `/docs/setup/production-environment/windows/intro-windows-in-kubernetes/` is _not_ minimum required content.

Oh, thanks for informing. I will keep that on hold for time being then :slightly_smiling_face:

**Comment by sftim** [comment]
It looks like all the minimum required content has a PR opened covering it in Hindi.

**Comment by kaiwalyakoparkar** [comment]
@sftim Could you please explain this point :sweat_smile: 
>help out with co-ordinating the remaining tasks to get Hindi added as an official localization

**Comment by anubha-v-ardhan** [comment]
Wow , we did it. Thanks Tim. : )

**Comment by vedant-kakde** [comment]
Now finally, we are done. 👍🏻

**Comment by Zain-Muiz** [comment]
Hey @anubha-v-ardhan , Can I take content/en/docs/tasks/tools/install-kubectl-windows ?

**Comment by vedant-kakde** [comment]
Hey Zain as of now we have completed localizing minimum requirements.

**Comment by verma-kunal** [comment]
Yay!! We did it 🎉
Great efforts everyone

**Comment by anubha-v-ardhan** [comment]
Hey @Zain-Muiz We are sorry but we don't have any pages left as of now. We are focusing on getting initial localization ready with minimum required content. We will soon expand this effort beyond minimum content once this is live on k8s.io.
In the meantime, if you are good with written Hindi, you can unofficially review PRs that have been made. : )
Thanks

**Comment by anubha-v-ardhan** [comment]
Folks, let's not do a lot of these commands.

**Comment by kaiwalyakoparkar** [comment]
Great job everyone :tada: 
Special shoutout to @verma-kunal

**Comment by verma-kunal** [comment]
> Great job everyone :tada: 
> Special shoutout to @verma-kunal for helping me understand the working and helping me cope up with this task :slightly_smiling_face: 

It was all your hard work & efforts 🙂✨

**Comment by vedant-kakde** [comment]
@anubha-v-ardhan I want to work on `content/en/docs/setup/production-environment/container-runtimes.md`

**Comment by anubha-v-ardhan** [comment]
@vedant-kakde The file you requested is already assigned to someone else : )

**Comment by vedant-kakde** [comment]
@anubha-v-ardhan I know but I think assignee is not active since last 3-4 days so I thought I can help in completing it.

**Comment by anubha-v-ardhan** [comment]
@vedant-kakde Its only been a few days, we can't say if she is inactive. However, you can ask her yourself to confirm. If she drops the file, we could assign it to you.
(Personally, I would recommend not asking folks in just a few days)

**Comment by vedant-kakde** [comment]
@anubha-v-ardhan ok no worries. I noticed some previous suggested changes were also not resolved which were suggested a week ago. We will wait for more days then.

**Comment by keshavcodex** [comment]
> i am taking `content/en/docs/setup/production-environment/landing` and it is a very long page so i will be taking more time.

it's done @anubha-v-ardhan, you can add this [PR here](https://github.com/kubernetes/website/pull/29518) on this page.

**Comment by anubha-v-ardhan** [comment]
Folks it would be nice if we all change our PR's base to `dev-1.22-hi.1` branch.
We can also remove our PRs from draft now.

cc: 
@rajatgupta24 @verma-kunal @kaiwalyakoparkar @vedant-kakde @keshavcodex @Darshnadas @ShivamTyagi12345 @arsheelsheikh @prashantpandey9

**Comment by anubha-v-ardhan** [comment]
## A brief summary of our progress as per steps mentioned in [Start a new localization](https://kubernetes.io/docs/contribute/localization/#start-a-new-localization):

### Find Community:
We have 10-12 active contributors. Our localization is self-sustaining. For communications on Slack, we have our dedicated channel [#kubernetes-docs-hi](https://kubernetes.slack.com/archives/CJ14B9BDJ)

### Join the Kubernetes GitHub Organization:
@anubha-v-ardhan @divya-mohan0209 and @mittalyashu are GitHub Organization members.

### Add your localization team in Github:
@anubha-v-ardhan @divya-mohan0209 and @mittalyashu are approvers and reviewers of `hi` localization. We have plans to add one more reviewer as @verma-kunal when he gets organization membership.

### Configure the workflow:
We have our own language label `language/hi`.

### Modify the site configuration:
Configuration block for `hi` is added to [config.toml](https://github.com/kubernetes/website/blob/main/config.toml).

### Add a new localization directory:
[content/hi](https://github.com/kubernetes/website/tree/main/content/hi) directory is added to this repo.

### Localize the community code of conduct:
Code of conduct has been localized and the PR has been merged [here in cncf/foundation](https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/hi.md) repo.

### Setting up the OWNERS files:
Our owners file is present here [content//hi/OWNERS](https://github.com/kubernetes/website/blob/main/content/hi/OWNERS).
@anubha-v-ardhan @divya-mohan0209 and @mittalyashu are currently `sig-docs-hi-owners` and `sig-docs-hi-reviews`. We have plans to add @verma-kunal to `sig-docs-hi-reviews` once he becomes organization member.

### Open a Pull Request:
We have the following PRs ready:
- Enable Hindi localization: #29674 
- Merge `dev-1.22-hi.1` to `main`: #29679
- Localization PRs mentioned in #29353 are being reviewed and merged to `dev-1.22-hi.1`

### Add a localized README file:
[README-hi.md] is added to [website repo](https://github.com/kubernetes/website).

### Launching your new localizaton:
We are aiming to get our localization with [minimum required content](https://kubernetes.io/docs/contribute/localization/#minimum-required-content) live before Kubernetes v1.23 release.

Thank you
cc: @sftim

**Comment by sftim** [comment]
> Configuration block for hi is added to config.toml.

(but disabled, which https://github.com/kubernetes/website/pull/29674 would address)

**Comment by harkiratsm** [comment]
@anubha-v-ardhan I wanted to work on ``` content/en/docs/tutorials/services/_index.md ``` also on ``` content/en/docs/tutorials/services/source-ip.md```

**Comment by anubha-v-ardhan** [comment]
@harkiratsm Thank you for wanting to contribute but our current aim is to get the localization live first with [minimum required content](https://kubernetes.io/docs/contribute/localization/#minimum-required-content). We will expand the localization to include other pages once bare minimum version goes live. 
Thank you for showing support. Hope you understand : )

**Comment by ShaileshKumar007** [comment]
Hi @anubha-v-ardhan . I want to ask if I can work on adding  `content/hi/releases` folder and `_index.md` , `notes.md` , `version-skew-policy.md`  files inside `releases` folder :-)

**Comment by anubha-v-ardhan** [comment]
@ShaileshKumar007 Thank you for wanting to contribute but our current aim is to get the localization live first with [minimum required content](https://kubernetes.io/docs/contribute/localization/#minimum-required-content). We will expand the localization to include other pages once bare minimum version goes live.
Thank you for showing support. Hope you understand : )

**Comment by anubha-v-ardhan** [comment]
cc @verma-kunal @rajatgupta24 @kaiwalyakoparkar @ShivamTyagi12345 

Can we have updates on the progress of localization work assigned to you please

@verma-kunal: `content/en/_index.md`
@kaiwalyakoparkar: `content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md`
@rajatgupta24: `content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md`
@ShivamTyagi12345: `content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md`

Thank you :)

**Comment by ShivamTyagi12345** [comment]
> cc @verma-kunal @rajatgupta24 @kaiwalyakoparkar @ShivamTyagi12345
> 
> Can we have update on the progress of localization work assigned to you please
> 
> @verma-kunal: `content/en/_index.md`
> @kaiwalyakoparkar: `content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md`
> @rajatgupta24: `content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md`
> @ShivamTyagi12345: `content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md`
> 
> Thank you :)

hey @anubha-v-ardhan ,Doing the final changes/checks. Can expect PR for all 7 files within 1 or 2 days

**Comment by verma-kunal** [comment]
> cc @verma-kunal @rajatgupta24 @kaiwalyakoparkar @ShivamTyagi12345
> 
> Can we have update on the progress of localization work assigned to you please
> 
> @verma-kunal: `content/en/_index.md`
> @kaiwalyakoparkar: `content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md`
> @rajatgupta24: `content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md`
> @ShivamTyagi12345: `content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md`
> 
> Thank you :)

Got a little busy with some work here!! Work in progress and would make a PR within 1-2 days 🙂

**Comment by kaiwalyakoparkar** [comment]
> cc @verma-kunal @rajatgupta24 @kaiwalyakoparkar @ShivamTyagi12345
> 
> Can we have update on the progress of localization work assigned to you please
> 
> @verma-kunal: `content/en/_index.md`
> @kaiwalyakoparkar: `content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md`
> @rajatgupta24: `content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md`
> @ShivamTyagi12345: `content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md`
> 
> Thank you :)

Was sick for the past couple of days. You can expect it in 1-2 days :)

**Comment by harkiratsm** [comment]
@anubha-v-ardhan can I work on ```website/content/en/docs/tasks/tools/_index.md``` ```website/content/en/docs/tasks/tools/install-kubectl-linux.md```

**Comment by verma-kunal** [comment]
> @verma-kunal you're working on https://deploy-preview-29679--kubernetes-io-main-staging.netlify.app/hi/ right? How's that going / need help with it?
> 
> (even if you have made little progress, I recommend opening a placeholder PR)

really sorry for the delay!! I'll be completing that today & making a PR :)

**Comment by PranshuSrivastava** [comment]
Hey, I would like to work on website/content/en/docs/test.md

**Comment by sftim** [comment]
@anubha-v-ardhan I recommend you omit some of the localization work from the issue description as I don't think it is minimum required content. Focusing on the minimum required content will bring forward the time when we can launch the Hindi localization officially.

The pages I'd omit _for now_ are:
- `content/hi/docs/setup/best-practices/cluster-large.md`
- `content/hi/docs/setup/best-practices/enforcing-pod-security-standards.md`
- `content/hi/docs/setup/best-practices/multiple-zones.md`
- `content/hi/docs/setup/best-practices/node-conformance.md`
- `content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md`
- `content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md`

(but, please, leave those issues open and encourage people to finish those useful PRs - the work is still valuable of course).

along with that, you should **add** entries to track a few extra pieces of minimum required content:
- `data/i18n/hi/hi.toml` (19 strings still missing)
- `content/hi/docs/reference/_index.md` (missing)
- `content/hi/docs/contribute/_index.md` (missing)

**Comment by anubha-v-ardhan** [comment]
Thanks @sftim I've done the needful

**Comment by sftim** [comment]
Also see PR #31077

**Comment by Shubh28698** [comment]
Hi @anubha-v-ardhan 

i would like to work on hindi localization of content/en/docs/contribute/analytics.md.

Also i was trying to merge PR, but it took too many commits and file changes even though i changed one file only. I have closed the PR . This is link to PR: https://github.com/kubernetes/website/pull/31251 . Sending this so you can have a look and guide me where i went wrong.

Can you please help me with it.

I have my localization already ready. Just stuck at this step

I am attaching SS that how i am making pull request. Please let me know if its right?
![Capture](https://user-images.githubusercontent.com/39756589/148657500-7f445993-0276-4c27-87c9-4367362916bc.PNG)

**Comment by sftim** [comment]
I'm guessing that #31987 is the last piece before launch - have I got that right?

**Comment by anubha-v-ardhan** [comment]
> I'm guessing that #31987 is the last piece before launch - have I got that right?

Yes @sftim !!

**Comment by sftim** [comment]
In https://github.com/kubernetes/website/pull/29679 I can see an [error](https://github.com/kubernetes/website/pull/29679#issuecomment-1120868315) logged. That will need fixing before we can merge the changes.

**Comment by anubha-v-ardhan** [comment]
Closing this issue
Hindi localization is live now with merge of #29679 
Great work everyone!

**Comment by sftim** [comment]
It's OK (and a good idea) to open a new tracking issue to cover future work on Hindi.

Is it OK to delete the branch _dev-1.22-hi.1_ now?

**Comment by verma-kunal** [comment]
> It's OK (and a good idea) to open a new tracking issue to cover future work on Hindi.

I totally agree with this!

> Is it OK to delete the branch _dev-1.22-hi.1_ now?

I believe if we'll directly be making PRs to the main branch, then we could delete. 
There was a conversation about making PRs from dev-1.22-hi.1 branch in batches of 5-10 localised pages to the main branch! 
Was any decision made on what approach we'll be following?!

**Comment by anubha-v-ardhan** [comment]
Let's discuss on Slack

**Comment by anubha-v-ardhan** [comment]
> It's OK (and a good idea) to open a new tracking issue to cover future work on Hindi.

Good idea @sftim but looks like folks have already started working under umbrella issues such as #36455 and #35600 making separate issues for each page.

**Comment by sftim** [comment]
/reopen

This _is_ the overall umbrella issue for the Hindi work.

**Comment by Kanika637** [comment]
As I opened the issue #40095, may I proceed with the specified work? @sftim

**Comment by sftim** [comment]
I recommend either closing #40095, or reshaping it to clearly track specific improvements.

---

This is open source. Anyone willing to honor our [values](https://kubernetes.io/community/#values) and abide by the [code of conduct](https://kubernetes.io/community/#conduct) is welcome to contribute. You don't need permission! You - and everyone else - are welcome to open a pull request.

See https://www.k8s.dev/docs/guide/ for more tips.

**Comment by sftim** [comment]
Also see https://github.com/kubernetes/website/issues/42653

**Comment by divya-mohan0209** [comment]
Hello @sftim , I don't think tracking progress on this issue is feasible any longer. As a cleanup measure, what I propose we do is:
- Close this issue.
- Open two fresh tracking issues - one for updating the documentation & one for introducing new documentation.
- Invite people to work off these two issues individually.

Does that sound okay by you? I ask, because you reopened this issue recently.

**Comment by sftim** [comment]
/area localization

I've no objection to this issue getting closed.

**Comment by seokho-son** [comment]
Hi folks! From the discussions in https://github.com/kubernetes/website/issues/29353#issuecomment-1844309110 and https://github.com/kubernetes/website/issues/29353#issuecomment-1876802163, it seems we've reached a general consensus on closing this issue. Considering that SIG-Docs is endeavoring to manage the quantity of open issues, I'll go ahead and close this issue.

/close
